tutorials/classical_simulation/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<p>State Preparation And Measurement (SPAM) errors were introduced in the introductory tutorial <code>Simulating sequences with errors and noises</code>. They will be further detailed in this notebook. We recommend you to check the introductory tutorial before going through this one.</p><p> </p><p>SPAM errors can be of two sorts:</p><p> </p><p>"incorrect identification of the final state of the atoms"</p><p> </p><p>allowing them to be trapped in the tweezers. This produces false negatives with rate ϵ'.</p><p> </p><p>Last line should be in the list with a bullet point</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>